celavi/pylca_celavi/concrete_life_cycle_inventory_editor.py

```diff
@@ -9,18 +14,29 @@
 """
 
 
-def concrete_life_cycle_inventory_updater(fd_cur2,yr,k,stage):
+def concrete_life_cycle_inventory_updater(fd_cur2, yr, k, stage):
+    # TODO: add docstrings to explain input variables and what the function
+    #  does.
+
+    # TODO: consider having sand_substitution_rate and coal_substitution_rate
```

Comment:
The coal and sand substitution rate are being obtained from the config file. PR #101